core/com.b2international.snowowl.server.console/src/com/b2international/snowowl/server/console/SnowOwlScriptingCommandProvider.java

```diff
@@ -64,9 +67,11 @@ private void run(CommandInterpreter interpreter) {
 		}
 		
 		try {
-			GroovyScriptHelper.reflectiveGroovyScriptExecutor(groovyScriptFile);
+			final String script = Files.lines(Paths.get(groovyScriptFile)).collect(Collectors.joining(System.getProperty("line.separator")));
+			final ClassLoader classLoader = ApplicationContext.getServiceForClass(ExtensionBasedRepositoryClassLoaderProviderRegistry.class).getClassLoader();
```

Comment:
This will throw a NPE, using the interface will fix it.

Reply:
Done!